ui/objects/semantic-ui/js/objects/editform/components/RestorationWork.jsx

```diff
@@ -26,68 +27,63 @@ export const RestorationWork = ({ activeIndex, handleActive, values }) => {
         "metadata.restorationWork.supervisors",
       ]}
       label="Práce"
+      styled
       active={activeIndex === 1}
       defaultActiveIndex={1}
       onClick={() => handleActive(1, values)}
     >
-      <div className="vert-div predmety__form__div-fields">
-        <div className="vert-div predmety__form__div">
+      <Grid columns={1}>
+        <Grid.Column>
           <TextField
             name="metadata.restorationWork.restorer"
             aria-label="Restauroval(a)"
             fieldPath="metadata.restorationWork.restorer"
-            className="form__input"
             label={
               <FieldLabel
                 htmlFor="metadata.restorationWork.restorer"
-                className="predmety__form__div__label"
                 label="Restauroval(a)"
               />
             }
           />
-        </div>
-        <div className="vert-div predmety__form__div">
+        </Grid.Column>
+        <Grid.Column>
           <RichInputField
             name="metadata.restorationWork.abstract"
             aria-label="Popis restaurování"
             fieldPath="metadata.restorationWork.abstract"
             label={
               <FieldLabel
                 htmlFor="metadata.restorationWork.abstract"
-                className="predmety__form__div__label"
                 label="Popis restaurování"
               ></FieldLabel>
             }
           ></RichInputField>
-        </div>
-        
-        <div className="vert-div predmety__form__div">
-          <ArrayField
-            addButtonLabel="Přidat vedoucího"
-            fieldPath="metadata.restorationWork.supervisors"
-            defaultNewValue={{ fullName: "", comment: "", institution: "" }}
-          >
-            {({ arrayHelpers, indexPath }) => {
-              const fieldPathPrefix = `${"metadata.restorationWork.supervisors"}[${indexPath}]`;
-              return (
+        </Grid.Column>
+        <ArrayField
+          addButtonLabel="Přidat vedoucího"
+          fieldPath="metadata.restorationWork.supervisors"
+          defaultNewValue={{ fullName: "", comment: "", institution: "" }}
+        >
+          {({ arrayHelpers, indexPath }) => {
+            const fieldPathPrefix = `${"metadata.restorationWork.supervisors"}[${indexPath}]`;
+            return (
+              <>
+                <Header as="h4">
+                  Vedoucí &nbsp;
+                  {values.metadata?.restorationWork?.supervisors?.[indexPath]
```

Comment:
Instead of this, get field value by `_get(values, fieldPathPrefix, ...)` just before const fieldPathPrefix & reuse value